pkg/builtinchecks/yamls/servicetype.yaml

```diff
@@ -0,0 +1,9 @@
+name: "forbidden-service-types"
```

Comment:
The check should be named something like "exposed-services" or something.

Reply:
renamed to `exposed-services`